src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.CommonSVTControls.ps1

```diff
@@ -722,26 +730,81 @@ class CommonSVTControls: ADOSVTBase {
+                            if ([Helpers]::CheckMember($_, "effectivePermissionValue")) {
+                                if ($this.excessivePermissionBitsForRepo -contains $_.effectivePermissionValue) {
+                                    $excessivePermissionsPerGroup += $_
+                                }
+                            }
+                        }   
+                        if ($excessivePermissionsPerGroup.Count -gt 0) {
+                            $groupFoundWithExcessivePermissions = $true
+                            # For PCBSA, resolve the group and check if PBS, PCBS are part of it
+                            if ($identity.displayName -like '*Project Collection Build Service Accounts') {
+                                $groupFoundWithExcessivePermissions = $false
+                                $url="https://dev.azure.com/{0}/_apis/Contribution/HierarchyQuery?api-version=5.1-preview" -f $($this.OrganizationContext.OrganizationName);                                
+                                $postbody=@'
+                                {"contributionIds":["ms.vss-admin-web.org-admin-members-data-provider"],"dataProviderContext":{"properties":{"subjectDescriptor":"{0}","sourcePage":{"url":"https://dev.azure.com/{2}/_settings/groups?subjectDescriptor={1}","routeId":"ms.vss-admin-web.collection-admin-hub-route","routeValues":{"adminPivot":"groups","controller":"ContributedPage","action":"Execute"}}}}}
+'@
+                                $postbody=$postbody.Replace("{0}",$identity.descriptor )
+                                $postbody=$postbody.Replace("{1}",$this.OrganizationContext.OrganizationName)
+                                $rmContext = [ContextHelper]::GetCurrentContext();
+                                $user = "";
+                                $base64AuthInfo = [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(("{0}:{1}" -f $user,$rmContext.AccessToken)))   
+                                try {
+                                    $response = Invoke-RestMethod -Uri $url -Method Post -ContentType "application/json" -Headers @{Authorization=("Basic {0}" -f $base64AuthInfo)} -Body $postbody
+                                    if([Helpers]::CheckMember($response.dataProviders.'ms.vss-admin-web.org-admin-members-data-provider', "identities"))
+                                    {
+                                        $buildServiceAccountIdentities = $response.dataProviders.'ms.vss-admin-web.org-admin-members-data-provider'.identities
+                                        foreach ($eachIdentity in $buildServiceAccountIdentities) {
+                                            if ($eachIdentity.displayName -like "*Project Collection Build Service ($($this.OrganizationContext.OrganizationName))" -or $identity.displayName -like "*Build Service ($($this.OrganizationContext.OrganizationName))") {
```

Comment:
$identity.displayName -like "*Build Service ($($this.OrganizationContext.OrganizationName))")  -> $eachIdentity?